### PR TITLE
docs: Document renovate testing strategy

### PR DESCRIPTION
**Pull request by joestringer**
Seems like this could be helpful to document for others attempting to
make changes to Renovate configuration.

### CODE REVIEW

File: Documentation/contributing/development/renovate.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#. `Enable the Renovate GitHub app <https://github.com/apps/renovate>`__ in
+   your GitHub account.
+#. Ensure that Renovate is enabled in the repository settings in the
+   `Renovate Dashboard <https://app.renovatebot.com/dashboard>`__.
+#. Trigger the Renovate app from the dashboard or push a fresh commit to your
+   fork's default branch to trigger Renovate again.
+#. Use the dashboard to trigger Renovate to create a PR on your fork and
+   validate that the proposed PRs are updating the correct parts of the codebase.
+
+Once you have tested that the renovate configuration works in your own fork,
```
Comment (ghost) [inline Documentation/contributing/development/renovate.rst]:
```suggestion
Once you have tested that the Renovate configuration works in your own fork,
```
nit

**Review by ghost — APPROVED** [review]
This is useful, thanks!

File: Documentation/contributing/development/renovate.rst (hunk 1 above)
Comment (joestringer) [reply Documentation/contributing/development/renovate.rst]: Fixed, thanks for the review.

**Review by aditighag — APPROVED** [review]
Thanks! I wasn't aware renovate changes could be tested on forks.

**Pull request merged**